test/unit/test_macro_calls.py

```diff
@@ -23,7 +23,7 @@ def setUp(self):
                     *
                     from {{ model }} )
                 {% endmacro %}""",
-            "{% macro test_my_test(model) %} select {{ dbt_utils.current_timestamp() }} {% endmacro %}",
+            "{% macro test_my_test(model) %} select {{ current_timestamp_backcompat() }} {% endmacro %}",
```

Comment:
Good catch.

For completeness, probably want to update references to `dbt_utils.current_timestamp` here too (although I don't know what the appropriate content would be):
- `core/dbt/clients/jinja_static.py`